1577: adds price bump config (#1639)
hyperledger/besu-docs: docs/public-networks/reference/cli/options.md

Set to `sequenced` to use the [sequenced transaction pool](../../concepts/transactions/pool#sequenced-transaction-pool).
The default is `sequenced` for the [enterprise/private profile](../../how-to/use-configuration-file/profile#enterpriseprivate-profile).

### `tx-pool-blob-price-bump`

<Tabs>

<TabItem value="Syntax" label="Syntax" default>

```bash
--tx-pool-blob-price-bump=<INTEGER>
```

</TabItem>

<TabItem value="Example" label="Example">

```bash
--tx-pool-blob-price-bump=25
```

</TabItem>

<TabItem value="Environment variable" label="Environment variable">

```bash
BESU_TX_POOL_BLOB_PRICE_BUMP=25
```

</TabItem>

<TabItem value="Configuration file" label="Configuration file">

```bash
tx-pool-blob-price-bump="25"
```

</TabItem>

</Tabs>

> Sets the price bump policy for re-issued blob transactions as a percentage increase in price. Note that a blob transaction can only replace, and be replaced by, a blob transaction. The default is `100%`.

Reviewer: I think remove the percent

```suggestion
Sets the price bump policy for re-issued blob transactions as a percentage increase in price. 
A blob transaction can only replace, or be replaced by, another blob transaction.
The default is `100`.
```

Reviewer: Otherwise looks good to me
